scripts/suite_objects.py

```diff
@@ -1598,7 +1639,7 @@ def register_action(self, vaction):
             # Add the missing dim
             vaction.add_local(self, _API_LOCAL, self.run_env)
             return True
-        # end if
+
```

Comment:
I think we want the original `# end if` here as per our conversation last week

Reply:
done.